_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,441 @@
+
+```html
+<head>
+	<title>Passed example 11 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p style="position:absolute; top:-9999em">Click "howdy" on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 12
+
+This paragraph includes the [visual reference word][] "right". The content in the second column is identified with the word "right" but also identified by referencing the word "howdy", thus matching the (**non-visual reference**) condition. Note that, despite the fact that the text is not included in the accessibility tree, this example is applicable because it is visible.
+
+```html
+<head>
+	<title>Passed example 12 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p aria-hidden="true">Click "howdy" on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 13
+
+This paragraph includes the [visual reference word][] "right". The user is told to find the navigation on the right and the navigation is also correctly identified by a `nav` element whose [accessible name][] contains the word "navigation", thus matching the (**non-visual reference**) condition.
+
+```html
+<head>
+	<title>Passed example 13 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the navigation on the right, for the non-essential links</p>
+		</div>
+		<nav>
+			<div class="col">
+				<ul>
+					<li>
+						<a href="https://www.w3.org/Consortium/contact">Contact</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Help/">Help and FAQ</a>
+					</li>
+				</ul>
+			</div>
+		</nav>
+	</div>
+</body>
+```
+
+#### Passed Example 14
+
+No [text node][] on this page includes any of the [visual reference words][].
+
+```html
+<p>Click the button, for a surprise</p>
+<button onclick="alert('Surprise!')">Howdy</button>
+```
+
+#### Passed Example 15
+
+The [text node][] inside the `p` element is not applicable because it is neither [visible][] nor [included in the accessibility tree][] due to its `display:none` styling. The [text node][] inside the `button` element does not include any of the [visual reference words][].
```

Comment:
Hence doesn't fulfill the applicability of this rule therefore shouldn't this example be in Inapplicable examples?

Reply:
Please consider my comment from passed example 2 for this example.

Reply:
I don't understand how that comment applies here. The discussion on Passed Example 2 is mostly on the use of "below" which is not present here.

In this example, the `p` element is hidden for all users. It effectively works as if it wasn't here:
```html
<button onclick="alert('Surprise!')">Howdy</button>
```
This does not fail the rule because there isn't any visual reference word used.

We could definitely improve the text inside the `p`, but I think it would actually weaken the example.
```html
<p style="display:none">Click the button labelled "howdy", for a surprise</p>
<button onclick="alert('Surprise!')">Howdy</button>
```
That would pass the rule whether the `display: none` is present or not. While in its current state, the example passes the rule **only** because `display: none` is here. I think that makes it a stronger example as it ensure that implementers are indeed ignoring hidden content.